### PR TITLE
Feature/initial component setup cr

### LINKED ISSUE #6: Implement React Router

### PR DESCRIPTION
**Pull request by Fordo29**
## What Features are Being Added?
Created initial Application using react framework

## What is Being Refactored?
Nothing as of yet

## Where should the reviewer start?
App.js and check all the import paths

## How should this be manually tested?
run npm start

## What are the relevant tickets?
NA

## Any Other Notes/Questions/Problems?
Not at this time

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,34 +10,11 @@
...
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <title>Book Club</title>
```
Comment (krdurrett) [inline public/index.html:14]: Thanks for changing the title!

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import '../styles/App.css';
+import React, { Component } from 'react';
+import BookContainer from './BookContainer';
+import Favorites from './Favorites';
+
+
+class App extends Component {
```
Comment (krdurrett) [inline src/components/App.js:7]: Great start to App

File: src/components/BookDetails.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import '../styles/BookDetails';
+
+class BookDetails extends Component {
```
Comment (krdurrett) [inline src/components/BookDetails.js:4]: Good idea making this a class since it will hold state.

File: src/index.js [hunk 4]
```diff
@@ -1,17 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from './components/App';
+import {BrowserRouter} from 'react-router-dom';
```
Comment (cagallo) [inline src/index.js:5]: Smart to add in Router from the beginning vs having to do it retroactively like we did in Rancid Tomatillos

File: src/components/BookContainer.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import '../styles/BookContainer.css';
+import BookCard from "./BookCard";
+
+const BookContainer = (props) => {
+
+    return(
+        <h2>Hi Ethan's books container</h2>
```
Comment (ectweitmann) [inline src/components/BookContainer.js:8]: Yes this is my books container. No one touch it please lol

**Review by ectweitmann — APPROVED** [review]

File: src/components/Favorites.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import '../styles/Favorites.css';
+
+class Favorites extends Component {
```
Comment (krdurrett) [inline src/components/Favorites.js:4]: Good idea making this a class since it will hold state (favorites)

**Review by cagallo — APPROVED** [review]
Looking great Christine, time to build this app!

**Review by krdurrett — APPROVED** [review]
Nice start to the component architecture

**Comment by ectweitmann** [comment]
Nice and atomic! Good work Christine.

**Pull request merged**